src/main/kotlin/nl/info/zac/documentcreation/converter/DocumentCreationDataConverter.kt

```diff
@@ -125,8 +132,10 @@ class DocumentCreationDataConverter @Inject constructor(
             )
         }
 
-    private fun createAanvragerDataNatuurlijkPersoon(bsn: String): AanvragerData? =
-        brpClientService.retrievePersoon(bsn)?.let { convertToAanvragerDataPersoon(it) }
+    private fun createAanvragerDataNatuurlijkPersoon(bsn: String, zaakNummer: String): AanvragerData? {
```

Comment:
```suggestion
    private fun createAanvragerDataNatuurlijkPersoon(bsn: String, process: String): AanvragerData? {
```
Is this the process?

Reply:
Yes slightly changed to process.

Reply:
The idea behind PROCESS was to construct the value expected by the X-Verwerking header for BRP protocollering, which is formed by combining a specific context (e.g. from a zaak or while creating a document or zaak) and the action (e.g. "Zaak aanmaken").

Reply:
@edgarvonk this value is not the `zaak nummer`, but an indication of the context of the process in progress for requesting the BRP details.

Maybe a better name is then `requestContext`.